pandas/compat/numpy/function.py

```diff
@@ -101,7 +102,8 @@ def validate_argmax_with_skipna(skipna, args, kwargs):
     return skipna
 
 
-ARGSORT_DEFAULTS = OrderedDict()
+ARGSORT_DEFAULTS = OrderedDict() \
+    # type: OrderedDict[str, Union[int, str, None]]
```

Comment:
Use Optional instead of Union None

Reply:
You mean like this, right?

    # type: OrderedDict[str, Optional[int, str]]

Reply:
@WillAyd I changed all `Union` to `Optional`, where `None` was in there.